### PR TITLE
Fixes #17947: Document how to enable/upgrade plugins after rudder upgrade

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17947

### CODE REVIEW

File: src/reference/modules/installation/pages/upgrade/notes.adoc [hunk 1]
```diff
@@ -9,6 +9,39 @@ xref:administration:procedures.adoc#_migration_backups_and_restores[backup proce
...
+== Plugins disabled during upgrade
+
+Each time Rudder is upgraded, plugins are disabled. You need to enable them 
+back after upgrade.
+For that, start by checking if update are available for you new version (it will
+be mandatory for a minor or major version upgrade), and enable them back:
+
+----
+
+rudder package update
```
Comment (amousset) [inline src/reference/modules/installation/pages/upgrade/notes.adoc:21]: This only work for people with a subscription, we need an alternate doc for other users.

**Comment by fanf** [comment]
PR updated with a new commit

File: src/reference/modules/installation/pages/upgrade/notes.adoc [hunk 2]
```diff
@@ -9,6 +9,52 @@ xref:administration:procedures.adoc#_migration_backups_and_restores[backup proce
...
+rudder package upgrade-all
+rudder package plugin enable-all
+
+----
+
+Without a subscription, proceed as you usually do get the last plugin package,
```
Comment (amousset) [inline src/reference/modules/installation/pages/upgrade/notes.adoc:30]:
```suggestion
Without a subscription, proceed as you usually do get the latest plugin package, 
```

File: src/reference/modules/installation/pages/upgrade/notes.adoc [hunk 3]
```diff
@@ -9,6 +9,52 @@ xref:administration:procedures.adoc#_migration_backups_and_restores[backup proce
...
+rudder package install-file /path/to/plugin.rpkg
+
+----
+
+
+You can also check for the state of all plugins:
+
+----
+
+rudder package list
```
Comment (amousset) [inline src/reference/modules/installation/pages/upgrade/notes.adoc:44]:
```suggestion
rudder package list --all
```

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**